core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/component/trace/componentTrace/HttpComponentQuery.java

```diff
@@ -0,0 +1,26 @@
+package io.metadew.iesi.metadata.definition.component.trace.componentTrace;
+
+import io.metadew.iesi.metadata.definition.Metadata;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class HttpComponentQuery {
+
+    @Builder
+    public HttpComponentQuery(UUID id, HttpComponentQueryKey httpComponentQueryID, String name, String value) {
```

Comment:
id should be of class HttpComponentQueryParameterTraceKey
HttpComponentQueryKey should be ComponentTraceKey